### DISCUSSION TITLE
Change color of logging levels

### DISCUSSION THREAD

**Discussion by jontobonto** [Textualize/rich#2169] [Q&A] [answered]
Hi,
I was wondering if there is a way to change these colors: (So from INFO, DEBUG, WARNING, etc)
![image](https://user-images.githubusercontent.com/67862800/161766764-58a69777-e24e-48c1-86c6-b837c7a9c8e1.png)

This is the basic conf:
```py
logging.basicConfig(format=r"[%(name)s] %(message)s", handlers=[RichHandler()])
```

**Comment by willmcgugan** [comment, marked as answer]
See docs on [theming](https://rich.readthedocs.io/en/latest/style.html#style-themes)

**Reply by jontobonto** [reply]
I have done it like this but it does not change the color of the WARNING message for me
```py
import logging
from rich.logging import RichHandler
from rich.theme import Theme
from rich.console import Console

_theme = Theme({"WARNING": "dark_orange"})
_console = Console(theme=_theme)

logging.basicConfig(format=r"[%(name)s] %(message)s", handlers=[RichHandler(console=_console)])

logger = logging.getLogger("main")
logger.setLevel(logging.DEBUG)

logger.debug("Hi")
logger.info("Hi")
logger.warning("Hi")
logger.error("Hi")
logger.critical("Hi")
```
![image](https://user-images.githubusercontent.com/67862800/161773934-a2c4747e-6920-491e-baae-fbfa704d90a4.png)

**Reply by willmcgugan** [reply]
Run `python -m rich.theme` and note the `logging.level.*` styles.

**Reply by jontobonto** [reply]
```py
_theme = Theme({"logging.level.warning": "dark_orange"})
```
That one worked! Thank you!
![image](https://user-images.githubusercontent.com/67862800/161776912-a274b61c-9a43-4f98-9d14-5fad5033bfc9.png)
